docs/content/engines/flink.md

```diff
@@ -210,3 +210,19 @@ All Flink data types are supported, except that
 
 * `MULTISET` is not supported.
 * `MAP` is not supported as primary keys.
+
+## Use Flink Managed Memory
+In order to improve the buffer pool for writers in Paimon tasks.Paimon tasks can create memory pools based on Executor Memory which will be managed by Executor, such as Managed Memory in Flink TaskManager. It will improve the stability and performance of sinks by managing writer buffers for multiple tasks through Executor.
+
+The following properties can be set if using the flink managed memory:
+
+| Option    | Default | Description                                                                                                                                                                    |
+|------------|----------|---------------------------------------------------------------------------------------------------------------------------------------------------------------|
+| sink.use-managed-memory-allocator | false | If true, flink sink will use managed memory for merge tree; otherwise, it will create an independent memory allocator.                   |
```

Comment:
```suggestion
| sink.use-managed-memory-allocator | false | If true, Flink sink will use managed memory for merge tree; otherwise, it will create an independent memory allocator.                   |
```

Also, what does "independent memory allocator" mean? An independent memory allocator for each parallelism? You might also want to stress that this allocator will use heap memory, not Flink managed memory.

Reply:
Thank you for the review. I have made some modifications to enrich some of the content